.github/workflows/build_image.yaml

```diff
@@ -19,6 +19,7 @@ env:
   TIMESCALE_STATIC_PRIMARY: "true"
   TIMESCALE_TSDB_ADMIN: "1.0.0"
   TIMESCALEDB_TOOLKIT_EXTENSIONS: 1.6.0 1.7.0 1.8.0 1.10.1 1.11.0
+  TIMESCALE_OSM_EXTENSION: "0.0.2"
```

Comment:
should we add 0.0.2 and 0.0.3 as a list?

Reply:
@sgichohi I changed it for installing every osm:* release for each PG version, do you think it worth to separate this as an external script (following toolkit pattern) or it's fine how it is now ?
 
https://github.com/timescale/timescaledb-docker-ha/blob/de5fe6c846229e4d93acdd5a1091b052b1b5ba43/Dockerfile#L303-L323

Reply:
cc @feikesteenbergen might know best.